applications/location-api/README.md

```diff
@@ -0,0 +1,161 @@
+
+
+[Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International Public License](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode)
+
+(CC) 2020 [LINKS Foundation](https://linksfoundation.com/)
+
+# MEC Location API simulator 
+
+- [MEC Location API simulator](#mec-location-api-simulator)
+	- [Application Description](#application-description)
+	- [Repository content](#repository-content)
+	- [Pre Requisites](#pre-requisites)
+	- [Installation](#installation)
+		- [Load docker images locally](#load-docker-images-locally)
+		- [Install the MEC Location API Service](#install-the-mec-location-api-service)
+		- [Start a simulation](#start-a-simulation)
+		- [Install consumer sample application](#install-consumer-sample-application)
+	- [How do I develop my MEC Application?](#how-do-i-develop-my-mec-application)
+
+Application Description
+---
+The Application is a Location API Simulator registered to OpenNESS platform as producer application. It provides a MEC Location Service accessible via Location API as specified in the ETSI GS MEC013 document, appearing as a RESTful web service. It has a Graphical User Interface enabling developers to simulate mobile users' movements by feeding the simulator with a GPS track in .gpx format.  
```

Comment:
It would be useful to hyperlink "ETSI GS MEC013" to the reference specs doc, or include in a *References* section to the bottom. It is ok to do both. This is essential for first time users who are not from a Telco background.

Reply:
Is the .gpx format industry-known standard? If not, please provide a reference.